--- www.openmicroscopy.org/community/style.php?id=7&lang=en.orig	2019-07-31 14:40:20.000000000 +0100
+++ www.openmicroscopy.org/community/style.php?id=7&lang=en	2019-07-31 15:04:53.000000000 +0100
@@ -3948,4 +3948,10 @@
 
 * html .ieVisible {display: block;}
 
-/* end ome.css */
\ No newline at end of file
+/* end ome.css */
+
+
+/* Manual overrides to hide non-static UI elements */
+form, ul.linklist.rightside, .quick-login, .buttons, #jumpbox~h3, #jumpbox~p, .headerspace~h3, .headerspace~p, ul.linklist li.rightside {
+  display: none;
+}
